<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
	<class>EditTaskWindowClass</class>
	<widget class="QDialog" name="EditTaskWindowClass">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>650</width>
				<height>330</height>
			</rect>
		</property>
		<property name="windowTitle">
			<string>EditTaskWindow</string>
		</property>
		<layout class="QGridLayout" name="gridLayout" columnstretch="1,3">
			<item row="6" column="1">
				<widget class="QLineEdit" name="qleImportantTask"/>
			</item>
			<item row="1" column="1">
				<widget class="QLabel" name="qlOldName">
					<property name="text">
						<string/>
					</property>
				</widget>
			</item>
			<item row="6" column="0">
				<widget class="QLabel" name="qlNewImportantTask">
					<property name="text">
						<string>Input a new the importance</string>
					</property>
				</widget>
			</item>
			<item row="7" column="1">
				<widget class="QLabel" name="qlOldImportant">
					<property name="text">
						<string/>
					</property>
				</widget>
			</item>
			<item row="0" column="0">
				<widget class="QLabel" name="qlnewNameTask">
					<property name="text">
						<string>Input a new name task</string>
					</property>
				</widget>
			</item>
			<item row="3" column="1">
				<widget class="QLabel" name="qlOldText">
					<property name="text">
						<string/>
					</property>
				</widget>
			</item>
			<item row="4" column="1">
				<widget class="QDateTimeEdit" name="qdteDate"/>
			</item>
			<item row="4" column="0">
				<widget class="QLabel" name="qlNewDateTask">
					<property name="text">
						<string>Input a new date</string>
					</property>
				</widget>
			</item>
			<item row="2" column="1">
				<widget class="QLineEdit" name="qleTextTask"/>
			</item>
			<item row="0" column="1">
				<widget class="QLineEdit" name="qleNameTask"/>
			</item>
			<item row="2" column="0">
				<widget class="QLabel" name="qlNewTextTask">
					<property name="text">
						<string>Input a new task text</string>
					</property>
				</widget>
			</item>
			<item row="5" column="1">
				<widget class="QLabel" name="qlOldDate">
					<property name="text">
						<string/>
					</property>
				</widget>
			</item>
			<item row="8" column="1">
				<layout class="QHBoxLayout" name="horizontalLayout">
					<item>
						<widget class="QPushButton" name="pbAccept">
							<property name="text">
								<string>Accept</string>
							</property>
						</widget>
					</item>
					<item>
						<widget class="QPushButton" name="pbReject">
							<property name="text">
								<string>Cancel</string>
							</property>
						</widget>
					</item>
				</layout>
			</item>
		</layout>
	</widget>
	<layoutdefault spacing="6" margin="11"/>
	<resources/>
	<connections/>
</ui>
